### ISSUE TITLE
List of all transactions related to account (private key)

### ISSUE THREAD

**Issue by ealymbaev** [Boilertalk/Web3.swift#71]
Is it possible to get the list of all past (historical) transactions quickly?
As I can see now - I can only get the count of transactions for account (using nonce number), but how can I get all these transactions in order to show to the user?

**Comment by Ybrin** [comment]
No, as far as I know neither geth nor parity provide this option. You will have to use a service like the etherscan api. There are versions of etherscan for all major testnets and the mainnet.

https://etherscan.io/apis
